src/index.html

```diff
@@ -1,11 +1,285 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>title</title>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+      integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Javascript Capstone</title>
 </head>
-<body>
-    <h1>Header 1</h1>
-</body>
-</html>
\ No newline at end of file
+<body class="bg-black">
+    <header class=" bg-dark">
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark" id="nav">
+            <div id="nav-div" class="container-fluid w-50">
+              <a class=" display-3  bg-dark ms-5 text-danger rounded-2" id="logo" href="#index.html">Netfetch</a>
+              <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+              </button>
+              <div class="collapse navbar-collapse ms-5 align-items-center" id="navbarNav">
+                <ul class="navbar-nav ms-4">
+                  <li class="nav-item">
+                    <a id="movies-button" class="nav-link active" aria-current="page" href="#">Movies</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="#">Categories</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="#">Years</a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </nav>
+
+    </header>
+    <!-- cards start -->
+    <main id="main" class="d-flex mt-3 p-3 gap-3 w-100 m-auto"></main>
+
+    <!-- Button trigger modal -->
+    <!-- <button
+      type="button"
+      class="btn btn-primary"
+      data-bs-toggle="modal"
+      data-bs-target="#exampleModal"
+    >
+      Click me
+    </button> -->
```

Comment:
- [ ] [OPTIONAL ] Kindly remove all unused lines of code in order for us to have a clean code-base.